encoders/lendianSteganography.py

```diff
@@ -0,0 +1,89 @@
+# Lendian stegonography system
+import random
+
+requiredParams = {
+        'encode':{
+            },
+        'decode':{
+            }
+        }
+
+def randomSound():
+    return random.randint(-32767, 32767)
+
+def encodeChar(char, encode_type='random'):
+    # Turn into an integer
+    char = ord(char)
+
+    # Convert to bits with padding to ensure 8 bit
+    bits = []
+    for i in range(8):
+        bits.append( char & 1 )
+        char = char >> 1
+
+    encoded_data = []
+    if encode_type == 'random':
+        for i in bits:
+            tmp_data = randomSound()
+            if i == 0:
+                tmp_data = tmp_data & ~1
+            else:
+                tmp_data = tmp_data | 1
+
+            encoded_data.append(str(tmp_data))
+
+    else:
+        #TODO: Is this how we are handling errors?
```

Comment:
Try raising an error that's more descriptive; remember, the user won't know which module had an issue, they'll just run a loooong command and then get this non-specific error message.